test.sh

```diff
@@ -302,4 +303,10 @@ if [[ "$RUN" =~ "godep-restore" ]] ; then
   end_context #godep-restore
 fi
 
+if [[ "$RUN" =~ "errcheck" ]] ; then
+  start_context "errcheck"
+  run_and_comment errcheck -ignore 'io:Write,Close,os:Remove,github.com/letsencrypt/boulder/Godeps/_workspace/src/gopkg.in/gorp.v1:Rollback,net/http:Write' -ignorepkg 'github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cactus/go-statsd-client/statsd,github.com/letsencrypt/boulder/log,github.com/letsencrypt/boulder/metrics,github.com/letsencrypt/boulder/rpc/pb' ./akamai/ ./bdns/ ./bin/ ./ca/ ./cmd/ ./core/ ./data/ ./log/ ./mail ./metrics ./mocks ./policy ./probs ./publisher ./ra ./reloader ./rpc ./sa ./test ./va ./wfe
```

Comment:
Since this ignore flag contains Rollback, why were the fixes to the various rollback calls necessary?

Also, why the explicit list of packages? Are some intentionally left out? I'd prefer to do something like the `go install` command, where we use `go list` to get a list of packages. If there are some we need to exclude for now, we can do that with a grep -v.